packages/tscc/src/log/spinner.ts

```diff
@@ -1,7 +1,14 @@
+/**
+ * @todo
+ * The `definedVar = undefined;` statements in this file are soul-crushing, but
```

Comment:
Huh, I see that this code pattern makes migrating to `--strict` compilation a bit painful. Those `definedVar`s actually represents a "state" of a spinner. One way to overcome this would be to introduce extra boolean flags, set `flag = false` in place of `definedVar = undefined`, and referencing that flag whenever we checked non-nullability of `definedVar`.